src/platforms/aarch64_linux_musl.rs

```diff
@@ -180,11 +183,14 @@ fn build_musl(build_directory: &Path, target_directory: &Path) -> anyhow::Result
 /// The `crt` building sequence.
 ///
 fn build_crt(
+    mut targets: Vec<String>,
     source_directory: &Path,
     build_directory: &Path,
     target_directory: &Path,
     use_ccache: bool,
 ) -> anyhow::Result<()> {
+    targets.insert(0, "AArch64".to_string());
```

Comment:
I nervous about the naked string constant (in addition to the duplicated code between here and src/platforms/x86_64_linux_musl.rs).

Reply:
> Outdated